stdlib/zipimport.pyi

```diff
@@ -22,3 +23,6 @@ class zipimporter(object):
     def get_source(self, fullname: str) -> str | None: ...
     def is_package(self, fullname: str) -> bool: ...
     def load_module(self, fullname: str) -> ModuleType: ...
+    if sys.version_info >= (3, 10):
+        def find_spec(self, fullname: str, target: None = ...) -> ModuleSpec | None: ...
```

Comment:
It seems wrong to disallow all non-`None` targets, because it doesn't do anything with the target at runtime. This is the annotation used for `target` in `sys.pyi`:

```suggestion
        def find_spec(self, fullname: str, target: ModuleType | None = ...) -> ModuleSpec | None: ...
```